media-libs/ilmbase/files/ilmbase-2.2.1-use-gnuinstalldirs.patch

```diff
@@ -0,0 +1,297 @@
+--- a/CMakeLists.txt	2017-11-17 23:00:23.000000000 +0000
```

Comment:
Should I ask how does a patch named `gnuinstalldirs` end up rewriting half of CMake, and adding a pkg-config file? Does this have a real perspective of being merged upstream or are you aiming for patch-forking it?

Reply:
Yes. It's scheduled to be merged upstream: https://github.com/openexr/openexr/pull/242